Dockerfile-test

```diff
@@ -0,0 +1,18 @@
+FROM continuumio/miniconda3:4.3.11
```

Comment:
Agreed. Copy/pasted from what we had in Dockerfile. Fun(ny) fact: the primary use of conda in the image is to install and then uninstall Node. What a world.